src/UrlGenerator/LocalUrlGenerator.php

```diff
@@ -4,15 +4,16 @@
 
 use Spatie\MediaLibrary\UrlGenerator\LocalUrlGenerator as SpatieLocalUrlGenerator;
 
-class LocalUrlGenerator extends SpatieLocalUrlGenerator {
+class LocalUrlGenerator extends SpatieLocalUrlGenerator
+{
+    public function getUrl(): string
+    {
+        if ($this->media->disk == 'media_private') {
```

Comment:
It will be better to use strict compare ===

Reply:
Sure, will be fixed